owasp_zap_historic/app.py

```diff
@@ -239,6 +239,84 @@ def dashboard(db):
     return redirect(url_for('redirect_url'))
 
 
+@app.route('/<db>/ehistoric', methods=['GET'])
+def ehistoric(db):
+    """dashboard page"""
+    cursor = mysql.connection.cursor()
+    use_db(cursor, db)
```

Comment:
Since these two lines are used seemingly always together perhaps you should modify 'use_db' to create the cursor and return it?

```
def use_db(db_name):	
    """method to switch db"""
    cursor = mysql.connection.cursor()
    cursor.execute("USE %s;" % db_name)
    return cursor
```

Then the two lines could simply be:
`cursor = use_db(db)`
each time its used